shell/platform/darwin/macos/framework/Source/FlutterCompositor.mm

```diff
@@ -69,10 +69,17 @@
     }
   }
 
-  [view.surfaceManager present:surfaces
-                        notify:^{
-                          PresentPlatformViews(view, layers, layers_count);
-                        }];
+  CFTimeInterval presentation_time = 0;
+
+  if (layers_count > 0 && layers[0]->presentation_time != 0) {
+    presentation_time = layers[0]->presentation_time / 1000000000.0;
```

Comment:
IIRC the single quote can be used as a separator in numeric literals as of C++14, this actually seems pretty useful here. Or a comment noting we're converting units of time here.